zadania/zadanie_14.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+
+def decimal_to_dms(decimal_str):
+    decimal_str = decimal_str.replace("°", " ")
+    parts = decimal_str.split()
+    
+    decimal = abs(float(parts[0]))
+    direction = parts[1]
+    
+    degrees = int(decimal)
+    minutes = int((decimal - degrees) * 60)
+    seconds = round(((decimal - degrees) * 60 - minutes) * 60, 2)
+    
+    if direction in ["S", "W"]:
+        degrees = -degrees

```

Comment:
Jeśli na wejściu jest określony kierunek świata, to już liczba nie musi mieć znaku.